core/schema/directory.go

```diff
@@ -120,6 +121,21 @@ func (s *directorySchema) Install() {
 				`This should only be used if the user requires that their exec processes be the
 				pid 1 process in the container. Otherwise it may result in unexpected behavior.`,
 			),
+		dagql.NodeFunc("dockerBuild", s.dockerBuild).
+			View(AfterVersion("v0.18.3")).
+			Doc(`Builds a new Docker container from this directory.`).
+			ArgDoc("dockerfile", `Path to the Dockerfile to use (e.g., "frontend.Dockerfile").`).
+			ArgDoc("platform", `The platform to build.`).
+			ArgDoc("buildArgs", `Build arguments to use in the build.`).
+			ArgDoc("target", `Target build stage to build.`).
+			ArgDoc("secretArgs", `Secrets to pass to the build.`,
```

Comment:
nit: the `secretArgs` here is different (shorter) than the one in `schema/container.go`. Just checking if that's intentional